modules/op-release-notes-1-2.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-2_{context}"]
+= Release notes for {pipelines-title} Technology Preview 1.2
+
+[id="new-features-1-2_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.2 is now available on {product-title} {product-version}. {pipelines-title} TP 1.2 is updated to support:
+
+* Tekton Pipelines 0.16.3
+* Tekton `tkn` CLI 0.13.1
+* Tekton Triggers 0.8.1
+* ClusterTasks based on Tekton Catalog 0.16
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.2.
+
+=== Pipelines
+
+* This release of {pipelines-title} adds support for a disconnected installation.
+//* The `PodSecurityPolicy` configuration was changed to enable {pipelines-title} in a restricted environment by running the tekton pipelines controller and tekton pipelines webhook as non-root users.
+//needs confirmation from Gabe if this needs to be added.
+* You can now use the `when` field, instead of `conditions`, to run a task only when certain criteria are met. The key components of `WhenExpressions` are `Input`, `Operator`, and `Values`. If all the `WhenExpressions` evaluate to `True`, then the task is run. If any of the `WhenExpressions` evaluate to `False`, the task is skipped.
+* Step statuses are now updated if a TaskRun is cancelled or times out.
+* Support for Git Large File Storage (LFS) is now available to build the base image used by `git-init`.
+* You can now use the `taskSpec` field to specify metadata, such as labels and annotations, when a task is embedded in a pipeline.
+* Cloud events are now supported by pipeline runs. Retries with `backoff` are now enabled for cloud events sent by the cloud event pipeline resource.
+* You can now set a default `Workspace` configuration for any workspace that a `Task` declares, but that a `TaskRun` does not explicitly provide.
+* Support is available for namespace variable interpolation for the `PipelineRun` namespace and `TaskRun` namespace.
+* Validation for `TaskRun` objects is now added to check that not more than one persistent volume claim workspace is used when a `TaskRun` is associated with an Affinity Assistant. If more than one persistent volume claim workspace is used, the `TaskRun` fails with a `TaskRunValidationFailed` condition. Note that by default, the Affinity Assistant is disabled in {pipelines-title}, so you will need to enable the assistant to use it.
```

Comment:
the `TaskRun` _process_ fails
(If "process" isn't the right word, you could also just use the generic "task run")